pkg/cli/debug_check_store_test.go

```diff
@@ -79,14 +79,12 @@ func TestDebugCheckStore(t *testing.T) {
 
 	// Introduce a stats divergence on s1.
 	func() {
-		cache := engine.NewRocksDBCache(10 << 20 /* 10mb */)
```

Comment:
Interesting. I trust that you're tracking this somewhere.

Reply:
> Interesting. I trust that you're tracking this somewhere.

Yep.